src/Tests/Microsoft.Diagnostics.Monitoring.OpenApiGen/StatusCodeStrings.cs

```diff
@@ -11,5 +11,6 @@ internal class StatusCodeStrings
     {
         public static readonly string Status400BadRequest = StatusCodes.Status400BadRequest.ToString(CultureInfo.InvariantCulture);
         public static readonly string Status401Unauthorized = StatusCodes.Status401Unauthorized.ToString(CultureInfo.InvariantCulture);
+        public static readonly string Status423TooManyRequests = StatusCodes.Status429TooManyRequests.ToString(CultureInfo.InvariantCulture);
```

Comment:
Should be `Status429TooManyRequests`

Reply:
I really hit push a little too quickly on my changes.